### ISSUE TITLE
Runtime inference with ROS

### ISSUE THREAD

**Issue by picjul** [torch-points3d/torch-points3d#744]
Hello guys!
I am developing ROS nodes to make point cloud inference starting from networks trained using `torch_points3d` and clouds captured with RealSense L515. Training and validation give good results with IoU and mACC values ​​around 95%.
Is there a way to perform inference at runtime without going through the test set (by directly input the clouds as `Data` object to the network)? Something like:
```
...
ros_cloud = convert_raw_data_to_data_object() # already done
net = load_net_model(checkpoint)
net.set_input(ros_cloud)
output = net.forward()
...
```
The `forward.py` module seems to only work from point clouds stored in files, processed and grouped into batches. Do you think that is possible to obtain a decent framerate?
Thanks!

**Comment by CCInc** [comment]
Hi!  Yes that's correct, you can treat the model as a normal pytorch model and feed data through yourself. All you have to do is:

```python
model = ...
model.eval()
data = Data(pos=pos)
model.forward(data)
output = model.get_output()
```

Let me know if it helps!

**Comment by CCInc** [comment]
I'm closing due to activity, feel free to reach out again if you need further help!
